Makefile

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
I think we don't need a Makefile here because this is a template, not a repo for customers to build or run.